### DISCUSSION TITLE
Multiple Progress Bars in One Model

### DISCUSSION THREAD

**Discussion by liblaf** [charmbracelet/bubbletea#649] [Q&A] [answered]
If I have multiple progress bars in one model, how can I update the corresponding progress model when `Update()` receives a `progress.FrameMsg`?

**Comment by meowgorithm** [comment, marked as answer]
Hi! Each `progress` instance will only only respond to its own `FrameMsg`s, so it's perfectly fine to simply flow all messages through a progress meter's `Update` method.
